src/parser.c

```diff
@@ -68,16 +68,19 @@ static struct config_src *parser_sources = NULL;
  */
 
 static struct config_src *add_config_source(pr_fh_t *fh) {
-  pool *p = pr_pool_create_sz(parser_pool, PARSER_CONFIG_SRC_POOL_SZ);
-  struct config_src *cs = pcalloc(p, sizeof(struct config_src));
+  pool *p;
+  struct config_src *cs;
 
+  p = pr_pool_create_sz(parser_pool, PARSER_CONFIG_SRC_POOL_SZ);
   pr_pool_tag(p, "configuration source pool");
-  cs->cs_next = NULL;
+
+  cs = pcalloc(p, sizeof(struct config_src));
```

Comment:
This is a stylistic cleanup; I prefer separating variable declaration from variable assignments due to function calls.